.github/workflows/build_and_test.yml

```diff
@@ -415,8 +415,12 @@ jobs:
         java-version: ${{ matrix.java }}
     - name: List Python packages (Python 3.9, PyPy3)
       run: |
-        python3.9 -m pip list
-        pypy3 -m pip list
+        python3.9 -m pip list -v
+        du -sh /usr/local/lib/python3.9/dist-packages
+        du -sh /usr/local/lib/python3.9/dist-packages/*
```

Comment:
Is this for debugging?

Reply:
yes, but what about keeping it to monitor the disk space?

Reply:
I think it's fine without it. Even disk space takes a lot here, there's nothing much we can do about them.